sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -143,9 +143,16 @@ def get_database(
         :returns: A :class:`Database` instance representing the retrieved database.
 
         """
+        if isinstance(database, Database):
+            id_value = database.id
+        elif isinstance(database, dict):
```

Comment:
I've been bit by the explicit check for `dict` in the past - anything that implements `Mapping`  would be good enough here, no?

Reply:
Yes I think so, just seeing if this pattern gets tests to pass. Not being able to run the emulator tests locally is a big dev bottleneck. I don't really like having to type check here at all but not sure what a good alternative is if the argument is really that general.